renderer/shaders/hlsl.glsl

```diff
@@ -43,22 +43,22 @@
 #endif
 
 $typedef float3 packed_float3;
-#define make_half float
-#define make_half2 float2
-#define make_half3 float3
-#define make_half4 float4
```

Comment:
Nice! Were we not using half before?

Reply:
As far as I can tell, yeah, these always actually used float!

Reply:
Nice catch!